src/main/resources/messages_en.properties

```diff
@@ -462,6 +462,9 @@ list.delete.not.folder=Item is not a folder
 list.delete.root.folder.not.allowed=Root folder can not be deleted
 list.delete.not.owner=You cannot delete a list that you do not own
 list.delete.folder.has.child=Unable to delete folder because it has items.
+list.ids.invalid=The following list id(s): {0} were not found in the system
+list.invalid=ListId is not a list
+list.locked=List is locked
```

Comment:
ListId is not a list can be replaced by: We have not found the list id {0} in the system
@lik-83